fe/fe-core/src/main/java/com/starrocks/sql/plan/ScalarOperatorToExpr.java

```diff
@@ -488,9 +489,20 @@ public Expr visitCall(CallOperator call, FormatterContext context) {
                             "",
                             ((ConstantOperator) call.getChild(0)).getBigint());
                     break;
-                case "session_id":
-                    callExpr = new InformationFunction(fnName,
-                            ((ConstantOperator) call.getChild(0)).getVarchar(), 0);
+                case "rand":
+                case "random":
+                case "uuid":
+                    callExpr = new FunctionCallExpr(call.getFnName(), new FunctionParams(false, List.of()));
+                    Preconditions.checkNotNull(call.getFunction());
+                    callExpr.setFn(call.getFunction());
+                    callExpr.setIgnoreNulls(call.getIgnoreNulls());
+                    break;
+                case "sleep":
+                    Expr child =  buildExpr.build(call.getChild(0), context);
+                    callExpr = new FunctionCallExpr(call.getFnName(), new FunctionParams(false, List.of(child)));
+                    Preconditions.checkNotNull(call.getFunction());
+                    callExpr.setFn(call.getFunction());
+                    callExpr.setIgnoreNulls(call.getIgnoreNulls());
```

Comment:
We can ensure all `nonDeterministicFunctions` can be rewritten in `ScalarOperatorToExpr` otherwise BE can be not compatible if the arguments's sizes are not expected.